README.md

```diff
@@ -1,6 +1,10 @@
-Zcash 1.0.0
+Zcash Classic 1.0.0
 ===========
```

Comment:
Founders be like...
![200 7](https://cloud.githubusercontent.com/assets/6740874/19914644/7fb513d4-a069-11e6-9b04-d5fb0d031675.gif)